### PR TITLE
2286 [BE] Prepare sql script to addd surdo/speech specialists into Contributor table #2286

### PR DESCRIPTION
**Pull request by koylubaevnt**
* Made migration script for load SPECIALIST
* Fix some problem in model Contributor: contribution field can be null

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/model/Contributor.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ data class Contributor(
     @Enumerated(EnumType.STRING)
     var type: ContributorType = ContributorType.DEVELOPER,
     var pictureUrl: String? = null,
-    var contribution: Long = 0,
+    var contribution: Long? = null,
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/model/Contributor.kt:35]: why did you brake Kotlinish stile?
the intention is to have not nullable types.

here it is logical to have Long type with 0 as default.

Comment (koylubaevnt) [reply src/main/kotlin/com/epam/brn/model/Contributor.kt:35]: Because in Database this value can be null. 
Will we replace to non nullable values?

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/model/Contributor.kt:35]: semantically it is non null value
let's replace in DB too. ?

Comment (koylubaevnt) [reply src/main/kotlin/com/epam/brn/model/Contributor.kt:35]: OK

Comment (koylubaevnt) [reply src/main/kotlin/com/epam/brn/model/Contributor.kt]: I fixed it. Please review again

**Review by ElenaSpb — APPROVED** [review]

**Pull request merged**